scripts/update-all.sh

```diff
@@ -0,0 +1,7 @@
+#! /bin/sh
+
+NEW_VERSION=$(npm show prisma@$1 version)
+echo "$NEW_VERSION" > .github/prisma-version.txt
+
+pnpm -rc --parallel exec "$(pwd)/scripts/update-version.sh $NEW_VERSION"
```

Comment:
Why string-replace of the version first in package.json, and then update the lockfile? Becuase it is faster?

Reply:
It is faster yes and also enabled `--parallel` which can run in any order, in which case, if you are in a workspace it won't matter if you ran `install`, it could be at the root or in a sub-project it won't a matter because versions are already updated everywhere (so mostly a monorepo handling thingy + performance)

Reply:
That should get a comment - I don't think that will be obvious in a few days/weeks/months/years.

Reply:
Added a comment